.gitignore

```diff
@@ -6,7 +6,7 @@ coverage/*
 .bundle
 rdoc/*
 pkg
-
+/.idea
```

Comment:
IMO, it's bad habit to add environment related ignores to projects.

easy enough to add 
```
[core]
        excludesfile = /$HOME/.gitignore_global
```
to your `~/.gitignore

note, that i don't think `$HOME` will be interpolated there

otherwise, we endup with ignores for sublime, atom, vim, emacs, win-super-dupper-edit, etc.